engine/src/main/java/org/terasology/rendering/dag/nodes/BlurNode.java

```diff
@@ -48,30 +49,22 @@
     private FBO outputFbo;
 
     /**
-     * Throws a RuntimeException if invoked. Use initialise(...) instead.
-     */
-    @Override
-    public void initialise() {
-        throw new RuntimeException("Please do not use initialise(). For this class use initialise(...) instead.");
-    }
-
-    /**
-     * Initializes the BlurNode instance.
+     * Constructs a BlurNode instance.
      *
      * @param inputFboConfig an FBOConfig instance describing the input FBO, to be retrieved from the FBO manager
      * @param outputFboConfig an FBOConfig instance describing the output FBO, to be retrieved from the FBO manager
      * @param fboManager the FBO manager from which to retrieve -both- FBOs.
      * @param aRadius the blur radius: higher values cause higher blur. The shader's default is 16.0f.
      * @param aLabel a String to label the instance's entry in output generated by the PerformanceMonitor
      */
-    public void initialise(FBOConfig inputFboConfig, FBOConfig outputFboConfig, BaseFBOsManager fboManager, float aRadius, String aLabel) {
+    public BlurNode(Context context, FBOConfig inputFboConfig, FBOConfig outputFboConfig, BaseFBOsManager fboManager, float aRadius, String aLabel) {
         this.fboManager = fboManager;
         this.blurRadius = aRadius;
         this.performanceMonitorLabel = aLabel;
         this.inputFboUrn = inputFboConfig.getName();
         this.outputFboUrn = outputFboConfig.getName();
 
-        setupConditions();
+        setupConditions(context);
```

Comment:
I'm not sure about the necessity of this change and in any case it would probably be good for a separate commit.